examples/siro_sandbox/hydra_helper.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from habitat.config.default_structured_configs import (
+    HabitatConfigPlugin,
+    register_hydra_plugin,
+)
+from habitat_baselines.config.default_structured_configs import (
+    HabitatBaselinesConfigPlugin,
+)
+
+
+def register_hydra_plugins():
```

Comment:
I'm not calling this yet, but I'm guessing I'll need when I switch to `@hydra.main` for the example HITL applications.